Packs/XSIAMCompliance_HIPAA/README.md

```diff
@@ -0,0 +1,9 @@
+## **Generate Compliance Dashboards and Reports**
+The compliance pack provides dashboards and reports for a variety of regulations. These dashboards and reports utilize queries which reference the XSIAM data model. Once data has been onboarded to XSIAM and datasets have been mapped to the data model, these dashboards and reports will query the data model to generate the dashboards and reports. Use these dashboards and reports as part of yearly compliance audits to show auditors how data is being tracked in XSIAM.
+
+**Note:** This pack is currently offered as a free beta for the Compliance module. However, in the future, this module will be sold separately.
```

Comment:
Should this be **Compliance pack** or **Compliance module**?

Same for all Readmes

Reply:
@nkanon What do you think?

Reply:
Hi @ShirleyDenkberg and @yucohen, checking with Hadar and updating.